torchmetrics/text/rouge.py

```diff
@@ -97,12 +95,13 @@ def __init__(
             process_group=process_group,
             dist_sync_fn=dist_sync_fn,
         )
+        # TODO: Add deprecated warning for newline_sep argument.
```

Comment:
Lets change the unused variables from
```
newline_sep: bool = False, 
```
to
```
newline_sep: Optional[bool] = None,
``` 
and then have a warning like this:
```suggestion
        if newline_sep is not None:
            warn.warning("Argument `newline_sep` is deprecated in v0.6 and will be removed in v0.7")
```